public/javascripts/fontawesome-all.min.js

```diff
@@ -1,5 +0,0 @@
-/*!
```

Comment:
I think we can also delete `/public/fonts/`?

Reply:
Done.